data/v.json

```
@@ -30789,6 +30830,21 @@
                 "synonyms": [
                     "masculinization"
                 ]
+            },
+            {
+                "id": "37b9739b1f",
+                "def": "the normal development of male sexual characteristics in a male",
```

This definition seems wrong from my research.

```suggestion
                "def": "the development of masculine characteristics in a female",
```

---

(I'm not good with github. Tell me if I'm supposed to do something to accept this change.)

---

There should be a button with "Accept suggested change" or you can just make the change and push it up. I just realized that 

>Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.

And that's not me, lol, so I'm not sure if we can even push / contribute. 😢